### DISCUSSION TITLE
Friction2d (Friction 2d) Showcase/Renders

### DISCUSSION THREAD

**Discussion by AlmostMikhail** [friction2d/friction#407] [Show and tell]
Showcase your friction renders here:

My Showcase #11 

https://github.com/user-attachments/assets/826e9a97-2541-4d31-8ec6-5e81e2472cd7

**Comment by AlmostMikhail** [comment]
https://github.com/user-attachments/assets/f0089c19-4804-425e-9069-cb3c8d4c3e0c


Showcase #12

**Comment by samytichadou** [comment]
Hi, let's share some friction personal tests :
Some random kde animations i made to get the hang on friction : 

https://github.com/user-attachments/assets/4abf4c95-141b-4437-ad66-5deb7b75b1b4


https://github.com/user-attachments/assets/fe4e637b-23d7-410f-8bb4-dc1ad15332e5

Here is the project file : 
[kde_logo_2.friction.zip](https://github.com/user-attachments/files/18497603/kde_logo_2.friction.zip)

**Comment by samytichadou** [comment]
A little gif for a friend (poorly drawn/animated, but it was in a hurry to congratulate him for his new born son :) )
It is a french play on words on his name.

https://github.com/user-attachments/assets/3d16ae8d-c68b-418b-b52d-7ec3b8ea0399

**Comment by samytichadou** [comment]
A jingle in for a web tv i made, the fly animation was created in blender grease pencil, compositing, texts... made in friction

https://github.com/user-attachments/assets/92053094-e615-457c-81a0-bea9eac41379

**Reply by AlmostMikhail** [reply]
Thanks for sharing

**Comment by samytichadou** [comment]
https://github.com/user-attachments/assets/b5774ed8-d534-4e5b-8d00-1798af69f64b

Some mograph test i made

Here is the friction project file : https://e.pcloud.link/publink/show?code=XZ8IldZJIA7G6DSzW0dOOxx9Dt3g0q4kMM7

**Comment by AlmostMikhail** [comment]
Render May 

https://github.com/user-attachments/assets/ea1d467c-3a7a-4717-bc9a-858b0b828d86

**Comment by AlmostMikhail** [comment]
Friction render June ![InShot_20250609_010201648.gif](https://github.com/user-attachments/assets/da667c1a-cf7d-402e-afa9-e9859146814b)

**Comment by samytichadou** [comment]
Some logo animation for my association made with friction and inkscape :)

https://github.com/user-attachments/assets/211aaf86-107e-4f5e-a06d-7ef0f45c6e6b

https://github.com/user-attachments/assets/072ca35d-7087-4415-80c6-7ecfc4fc5923

**Comment by ezechuka** [comment]
I've been learning friction/inkscape for the past 3wks and i managed to create this 3sec clip :-/

https://github.com/user-attachments/assets/09660b94-e7af-4f1f-8fbf-27bcd11e2d6f
